packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -337,39 +344,45 @@ class EnrollVerbHandler extends AbstractVerbHandler {
     }
   }
 
-  Future<AtData> _fetchEnrollmentDataFromKeyStore(
-      String enrollmentKey, currentAtSign, String? enrollmentId) async {
-    AtData enrollData;
-    // KeyStore.get will not return null. If the value is null, keyStore.get
-    // throws KeyNotFoundException.
-    // So, enrollData will NOT be null.
-    try {
-      enrollData = await keyStore.get('$enrollmentKey$currentAtSign');
-    } on KeyNotFoundException {
-      throw AtEnrollmentException(
-          'enrollment id: $enrollmentId not found in keystore');
-    }
-    // If enrollment is not active, throw AtEnrollmentException
-    if (!SecondaryUtil.isActiveKey(enrollData)) {
-      throw AtEnrollmentException('The enrollment $enrollmentId is expired');
+  void _validateEnrollmentValidity(
+      EnrollStatus enrollStatus, String enrollmentId, Response response) {
+    if (EnrollStatus.expired == enrollStatus) {
+      response.isError = true;
+      response.errorCode = 'AT0028';
+      response.errorMessage =
+          'enrollment_id: $enrollmentId is expired or invalid';
+      return;
```

Comment:
Just a minor comment, Please remove the "return" statement. It is of no use.

Reply:
changed as part of [25eb563](https://github.com/atsign-foundation/at_server/pull/1536/commits/25eb563f94d1815b2af8171185a22e3496ded925)